src/OpenTK.Windowing.Desktop/NativeWindow.cs

```diff
@@ -1380,22 +1386,10 @@ public bool IsMouseButtonReleased(MouseButton button)
         /// <see cref="OpenTK.Windowing.Desktop.Monitors.GetMonitorFromWindow(NativeWindow)"/>
         /// to find it.
         /// </remarks>
+        [Obsolete("Use Monitors.GetMonitorFromWindow instead", true)]
         public unsafe MonitorHandle FindMonitor()
```

Comment:
When this function was written the monitor info class didn't exist. Perhaps we should consider not obsoleting it but replacing it with a simple call to the backing API (unless it exists and I missed it in due to viewing the diff.)

Reply:
Fair enough, can fix that the next time I have some time.

Reply:
Thinking again, this should stay obsolete, but have `iserror = false`. Over the next minor release it should be replaced with a method returning `MonitorInfo` rather than the handle.